index.html

```diff
@@ -321,19 +321,220 @@ <h4 class="pricing-table__subtitle-secondary my-0 pb-1">Subtitle second line goe
+                        </div>
+                    </div>
+                </div>
+                
+                <!-- Column 3 -->
+                <div class="col-xl-3 col-lg-4 col-md-6 col-sm-10 pt-11 footer-item">
+                    <div class="footer-item__email pb-9 pb-xs-9 d-flex flex-column gap-3">
+                        <h1 class="m-0 pb-4">Email us</h1>
+                        <div class="footer-item__email-name d-flex pt-1">
+                            <input type="text" name="name" id="name" placeholder="Your name" class="py-2 pl-3 pr-5">
+                            <label for="name"> <img src="/images/logo_user.png" alt=""></label>
+                            </div>
+                        <div class="footer-item__email-mail d-flex">
+                            <input type="email" name="email" id="email" placeholder="Your email" autocomplete="off" class="py-2 pl-3 pr-5">
+                            <label for="email" class="d-flex align-items-center"> <i class="fa-solid fa-envelope"></i></label>
+                        </div>
+                        <textarea class="footer-item__email-message py-2 px-3" name="message" id="message"
+                            title="message" placeholder="Your message"></textarea>
+                            
+                            <button class="footer-item__email-button py-2-plus">Send message</button>
+                    </div>
+
+                    <div class="footer-item__projects">
+                        <h1 class="m-0 pb-6">Projects / clients ?</h1>
+                        <div class="footer-item__projects-list d-flex flex-wrap justify-content-center pt-1 pb-2">
+                            <div class="footer-item__project"></div>
+                            <div class="footer-item__project"></div>
+                            <div class="footer-item__project"></div>
+                            <div class="footer-item__project"></div>
+                            <div class="footer-item__project"></div>
+                            <div class="footer-item__project"></div>
+                        </div>
+                        <div class="footer__indicators d-flex gap-2 justify-content-center pt-1">
+                            <div class="footer__indicator footer__indicator--current"></div>
+                            <div class="footer__indicator"></div>
+                            <div class="footer__indicator"></div>
+                        </div>
+                    </div>
+                </div>
+                
+                <!-- Column 4 -->
+                <div class="col-xl-3 col-lg-4 col-md-6 col-sm-10 pt-11 footer-item">
+                    <div class="footer-item__flickr">
+                        <h1 class="m-0 pb-6">Flickr feed</h1>
+                        <div class="footer-item__flickr-list d-flex flex-wrap justify-content-center pt-1">
+                            <div class="footer-item__flickr-feed"></div>
```

Comment:
Theo design màn to nhất thì nó có kích thước lớn nhất là 80*80 nhá. Em đang không để nó là hình vuông.